src/domain/invoice-expiration.ts

```diff
@@ -0,0 +1,20 @@
+import moment from "moment"
+
+const DEFAULT_EXPIRATIONS = {
+  BTC: { value: 1, unit: "days" as moment.unitOfTime.DurationConstructor },
+  USD: { value: 2, unit: "minutes" as moment.unitOfTime.DurationConstructor },
+}
+
+export const invoiceExpirationForCurrency = (
```

Comment:
I think this was just a proof of concept, what you really need to do is query the dealer an get a quote with the time (available) and value in sats for the give amount in fiat currency

Reply:
yes this will be a service call